pkgs/top-level/all-packages.nix

```diff
@@ -25164,6 +25164,10 @@ in
 
   magnetico = callPackage ../applications/networking/p2p/magnetico { };
 
+  mastodon-archive = callPackage ../tools/backup/mastodon-archive {
+    python = python3;
+  };
```

Comment:
```suggestion
  mastodon-archive = callPackage ../tools/backup/mastodon-archive { };
```

Reply:
This should also be in the `TOOLS` section.

Reply:
Done